sycl/plugins/unified_runtime/ur/adapters/level_zero/ur_level_zero.cpp

```diff
@@ -390,3 +390,57 @@ ZER_APIEXPORT zer_result_t ZER_APICALL zerPlatformGet(
 
   return ZER_RESULT_SUCCESS;
 }
+
+ZER_APIEXPORT zer_result_t ZER_APICALL zerPlatformGetInfo(
+    zer_platform_handle_t Platform, ///< [in] handle of the platform
+    zer_platform_info_t ParamName,  ///< [in] type of the info to retrieve
+    size_t *pSize, ///< [in,out] pointer to the number of bytes needed to return
+                   ///< info queried. the call shall update it with the real
+                   ///< number of bytes needed to return the info
+    void *ParamValue ///< [out][optional] array of bytes holding the info.
+                     ///< if *pSize is not equal to the real number of bytes
+                     ///< needed to return the info then the
+                     ///< ::ZER_RESULT_ERROR_INVALID_SIZE error is returned and
+                     ///< pPlatformInfo is not used.

```

Comment:
I am not against it but is this some known format of commenting API? I think we should use capital case at the beginning of the sentences.

Reply:
This is the format used in parent https://github.com/oneapi-src/unified-runtime/blob/cdb62e54bba8f9e56d3438e0d3d91e3037862f55/include/ur_api.h#L3709

I'll let the new owners choose/reformat as needed
The Unified Runtime is to become an external component for SYCL